<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_Background">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Range Shirley</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Limited Range Shirley</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tougaard</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_FBackground">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Background</string>
   </property>
  </widget>
  <widget class="MplWidget" name="MplWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ShowBackground">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Background</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_SubtractBackground">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract Background</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Background Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Done">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done!</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Background_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Range Shirley</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Limited Range Shirley</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tougaard</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Second Background</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Range1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Range2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Range</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
